backend/functions/tests/flashcard.test.js

```diff
@@ -223,27 +235,136 @@ describe("Testing Flashcard Set Functions", () => {
     });
 
     test("throws error updating flashcard set we don't own", async () => {
-      // Create a test flashcard set for error cases
+      const context = {auth: {uid: user.uid}};
+      // Attempt to update flashcard set
+      const updateFlashcardSet = fft.wrap(myFunctions.updateFlashcardSet);
+      await expect(updateFlashcardSet({flashcardId: unownedFCSetId, ...unownedFCSetData}, context)).rejects.toEqual(
+          new Error("You do not have permission to edit this flashcard set."),
+      );
+
+      // Validate flashcard set still exists
+      const doc = await admin.firestore().collection("flashcards").doc(unownedFCSetId).get();
+      expect(doc.exists).toBeTruthy();
+    });
+  });
+
+  //* **************** Test deleteFlashcardSet ******************* */
+  describe("deleteFlashcardSet", () => {
+    let delDocId;
+
+    beforeAll(async () => {
       const data = {
-        creatorId: "some-uuid",
-        title: "Unowned Flashcard set",
+        creatorId: user.uid,
```

Comment:
Are keeping the `creatorId` or not? cause there have been a lot of back and forth on that. i dont where we stand right now. and its not in the metadata.